docs/getting-started/index.md

```diff
@@ -24,3 +25,36 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
+
+There are many ways to get started with Apache Flink. Which one is the best for you depends on your goal and prior experience.
+
+### Taking a first look at Flink
+
+The **Docker Playgrounds** provide sandboxed Flink environments that are set up in just a few minutes and which allow you to explore and play with Flink.
+
+* The [**Operations Playground**](./docker-playgrounds/flink_cluster_playground.html) shows you how to operate streaming applications with Flink. You can experience how Flink recovers application from failures, upgrade and scale streaming applications up and down, and query application metrics.
+
+<!-- 
+* The [**Streaming SQL Playground**]() provides a Flink cluster with a SQL CLI client, tables which are fed by streaming data sources, and instructions for how to run continuous streaming SQL queries on these tables. This is the perfect environment for your first steps with streaming SQL. 
+-->
+
+### First steps with one of Flink's APIs
+
+The **Code Walkthroughs** are the best way to get started and introduce you step by step to an API.
+A walkthrough provides instructions to bootstrap a small Flink project with a code skeleton and shows how to extend it to a simple application.
+
+<!-- 
+* The [**DataStream API**]() code walkthrough shows how to implement a simple DataStream application and how to extend it to be stateful and use timers.
+-->
+* The [**DataStream API**](./tutorials/datastream_api.html) tutorial shows how to implement a basic DataStream application. 
```

Comment:
```suggestion
* The [**DataStream API**](./tutorials/datastream_api.html) tutorial shows how to implement a basic DataStream application. The `DataStream` API is the main abstraction in Flink to implement streaming use cases. It is the common basis for all other APIs of Flink, and provides a lot of flexibility.
```

Reply:
I'm suggestion suggesting this, because people won't necessarily know what the DataStream API is .